perfdash/metrics-downloader.go

```diff
@@ -182,13 +177,24 @@ func getBuildData(result JobToCategoryData, prefix string, category string, labe
 	return result[prefix][category][label]
 }
 
-type bucketUtil struct {
+// MetricsBucket is the interface that fetches data from a storage service.
+type MetricsBucket interface {
+	GetBuildNumbersFromBucket(job string) ([]int, error)
```

Comment:
We may eventually want to refactor these names. Bucket.GetFooFromBucket is redundant compared do Bucket.GetFoo(). 

I'd also love to see these go classes organized into a /pkg folder, with perhaps a metrics_bucket.go that owns this interface.

Reply:
Refactoring the names makes total sense and is an easy fix so I went ahead and did that. +1 on organizing the go classes; I would be open to refactoring some more in the future for better organization/readability.